src/Settings/settings.json

```
@@ -16,7 +16,7 @@
     {
         "name": "Editor Font",
         "type": "QFont",
-        "default": "QFont(\"monospace\")",
+        "default": "[](){ QFont font = QFontDatabase::systemFont(QFontDatabase::FixedFont); font.setPointSize(12); font.setStyleHint(QFont::TypeWriter); return font; }()",
```

```suggestion
        "default": "[](){ QFont font = QFontDatabase::systemFont(QFontDatabase::FixedFont); font.setStyleHint(QFont::TypeWriter); return font; }()",
```

We don't need to set the size. The bug only happens when it's created by `QFont::QFont(const QString &family)`, not when it's created by `QFontDatabase::systemFont(QFontDatabase::FixedFont)`.

---

Nope, this will change the font size to 9pt. Default font size is 12pt isn't it?

---

> Nope, this will change the font size to 9pt. Default font size is 12pt isn't it?

That's because the fixed-width font of your OS is 9pt. You can change your system font and see.

---

Okay. By default I mean default font size for application, not system default. I though default application font size for editor is 12pt. Anyway if you're considering os then your changes are fine.